pkg/rslog/logger_impl.go

```diff
@@ -187,53 +167,124 @@ func (l logrusEntryWrapper) WithFields(fields Fields) Logger {
 	return logrusEntryWrapper{Entry: e}
 }
 
-func (l logrusEntryWrapper) SetReportCaller(flag bool) {
-	l.Logger.SetReportCaller(flag)
-}
```

Comment:
I think this change could impact our debug logs, we use `SetReportCaller` for debug logs to be able to come with the context of the caller (source code file and line num). We might want to keep this, but being a single instance in use, this will impact other methods as well like `Infof/Warnf` and not just `Debugf/Tracef`

Reply:
I am OK if we abandon source file/lineno information from debug logs.

Reply:
Abandoning file/lineno information is probably more important given that we're headed in the direction of making it customer facing. Gives up information about the underlying source that we might not want to.

Reply:
@scottmmjackson And now I see I totally misread your comment. 😆 You're right, that can be a small risk!